### ISSUE TITLE
Hyperparameter mismatches between main.py and Appendix section B

### ISSUE THREAD

**Issue by sravan-ankireddy** [Time-MoE/Time-MoE#93]
Is it possible to provide the exact configuration used for training the models? A few hyperparams such as learning rate and warmup steps are described in the paper but some of them are different from the defaults set in main.py. 

It would be greate if you can either update all the defaults to the best values or provide a detailed command for launching the training script. We are trying to reproduce the zero-shot results for the 50m model but so far the performance of the model we trained is much worse than what I am seeing by using the provided chackpoint.

**Comment by Maple728** [comment]
Thank you for your interest in our work!

The default hyperparameters in main.py are configured for the fine-tuning scenario. The hyperparameters used for pretraining are detailed in our paper.

If you have any questions, please feel free to contact us.

**Comment by sravan-ankireddy** [comment]
Thanks for the response and clarifying the hyperparameter settings! 

In that case, I am not sure why I am not able to reproduce the results from the paper. The only other thing that's missing is the packing module. The paper mentions using sequence packing during training but the code does not use packing. By default it's set to use the non-packing fixed length sequence generation with padding. 

Could you please clarify this? Really appreciate the help here!
